charmhelpers/contrib/storage/linux/ceph.py

```diff
@@ -341,7 +340,8 @@ def _post_create(self):
         Do not add calls for a specific pool type here, those should go into
         one of the pool specific classes.
         """
-        if self.nautilus_or_later:
+        nautilus_or_later = cmp_pkgrevno('ceph-common', '14.2.0') >= 0
```

Comment:
Would it make more sense to have `nautilus_or_later` a property of the BasePool class, then you don't have to repeat the `cmp_pkgrevno` code ?